accepted/0000-remove-dependency-on-fulcio.md

```diff
@@ -0,0 +1,69 @@
+# RFC for linking packages to their source and build without using Sigstore Fulcio
+
+## Summary
+
+As it is showed in a recent paper that Sigstore Fulcio is an unnecessary trusted third party for linking npm packages to their source and build, we suggest the npm team consider the new approach presented in that paper which can be found at:
+
+https://eprint.iacr.org/2023/003
+
+The new approach not only eliminates the unnecessary dependency on an external black-box CA but also accelerates the signing process.
+
+## Motivation
+
+Last year an RFC for linking npm packages to their source and build using Sigstore Fulcio was proposed and accepted:
```

Comment:
to be clear, that RFC was merged, but never was accepted in an RFC call per the npm team's process.

Reply:
Yes, it is just a modification of the mentioned RFC and does not change the workflows except for removing the unnecessary dependency on Sigstore Fulcio.

Sorry I did not realize the mentioned RFC, though merged to the accepted folder, was never accepted. Some related code found in the cli/provenance branch also made me feel so. Thank you for letting me know.

Reply:
Hi @ljharb @ycchang I am new Engg Mgr for npm cli and my team also works on part of nm registry,

I want to reconfirm that the RFC was accepted and merged after thourough review and sicussions with npm team.

We do agree and respect open source community feedbacks and there can be disagreements, sometime we would like to move fast and course correct progressively based on all inputs (ship to learn), 

we would thouroughly reiview and discuss this RFC, consider the inputs and feedback and if needed would course correct on previous rfcs

Reply:
@saquibkhan that RFC was the first time that an RFC was accepted outside of an RFC call, and this one in particular was *not* thoroughly discussed, because there were no RFC calls (due to the npm team being heads down on v9). It sends a very negative message that erodes the trust built up with the community over years to accept an RFC in this way.